jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainPatch.java

```diff
@@ -56,6 +56,8 @@
 import java.nio.FloatBuffer;
 import java.util.HashMap;
 import java.util.List;
+import java.util.logging.Level;

```

Comment:
It looks like `import java.util.logging.Level;` is not used? if so please remove the unused import.